.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,13 +8,24 @@ assignees: ''
 ---
 
 **Is your feature request related to a problem? Please describe.**
+
 A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
 
 **Describe the solution you'd like**
+
 A clear and concise description of what you want to happen.
 
 **Describe alternatives you've considered**
+
 A clear and concise description of any alternative solutions or features you've considered.
 
 **Additional context**
+
 Add any other context or screenshots about the feature request here.
+
+### Checklist:
+
+- [ ] I have double checked that the issue does not currently exist. 
+- [ ] I have filled out the above template.
```

Comment:
I suggest we remove "I have filled out the template" from the checklist as going through the checklist means they are filling it out